starters/deno-oak-denodb/src/db/model/technology.ts

```diff
@@ -0,0 +1,16 @@
+import { DataTypes, Model } from '../../../deps.ts';
+
+export class Technologies extends Model {
```

Comment:
Shouldn't we be using a `User` model here?

Reply:
Or if we have Technologies in other starter kit, we can use that, sure.

Reply:
Based on the Document Dustin shared, this was the model he said the backend kits should have

Reply:
Ah, okay. I missed that. 🙊